tools/minicluster/minicluster.go

```diff
@@ -255,7 +279,6 @@ func newConfig(
 	cfg.Genesis.Blockchain.NumSubEpochs = 2
 	cfg.Genesis.Blockchain.NumDelegates = numNodes
 	cfg.Genesis.Blockchain.TimeBasedRotation = true
-	cfg.Genesis.Delegates = cfg.Genesis.Delegates[3 : numNodes+3]
-
+	cfg.Genesis.Delegates = cfg.Genesis.Delegates[3 : 3+8]
```

Comment:
numNodes is supposed to be the number of delegates. So we'd better not to hard code 8.